app/forms/settings/multi_lang_form.rb

```diff
@@ -59,7 +59,6 @@ def initialize(name:, current_language:)
         disabled: setting_disabled?(name),
         visually_hide_label: true,
         rich_text_options: {
-          text_area_id: "settings-#{name}",
```

Comment:
We're not using `MultiLangForm` currently, so I haven't been able to test this change.